Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -2740,6 +2743,12 @@ private void LynchCycle()
                         if (lynched.PlayerRole == IRole.Prince & !lynched.HasUsedAbility) //can only do this once
                         {
                             SendWithQueue(GetLocaleString("PrinceLynched", lynched.GetName()));
+                            var ScapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
+                            if (ScapeGoat != null)
+                            {
+                                SendWithQueue(GetLocaleString("ScapeGoatLynched", lynched.GetName()));
+                                KillPlayer(lynched, KillMthd.Lynch, killers: Players.Where(x => x.Choice == lynched.Id), isNight: false);
```

Comment:
Logical error... At this point, `lynched` is the prince. What you're doing is saying that if there is a scape goat alive, the prince dies.
```suggestion
                                SendWithQueue(GetLocaleString("ScapeGoatLynched", scapeGoat.GetName()));
                                KillPlayer(scapeGoat, KillMthd.Lynch, killers: Players.Where(x => x.Choice == lynched.Id), isNight: false);
```
It is arguable who the scape goat's killers are in this case but the people who lynched the prince do make sense.

The other question is: Should the scape goat really die if the prince got lynched? The whole point in the prince role is that if the prince is lynched, the village realizes their mistake and refrains from lynching anyone that day.